targets/TARGET_STM/TARGET_STM32L4/mbed_overrides.c

```diff
@@ -32,6 +32,10 @@ void mbed_sdk_init()
 {
     // Update the SystemCoreClock variable.
     SystemCoreClockUpdate();
```

Comment:
@c1728p9 I've now spent some time investigating everything being done in the STM32 HAL_Init() and clock setting functions, and while I think it's a little wasteful doing the clock detection twice, the HAL RCC clock setting functions actually make use of the HAL tick timer for checking timeouts when starting oscillators and such, so it is safer to keep the extra SystemCoreClockUpdate() call as you have it to ensure these clock setup timeouts are of the proper duration.  So, from my point of view, your code is okay as is.